packages/gatsby-plugin-typescript/README.md

```diff
@@ -36,6 +36,7 @@ module.exports = {
 ```
 
 For more detailed documentation on the available options, visit https://babeljs.io/docs/en/babel-preset-typescript#options.
+To add TypeScript Babel plugins (e.g. `@babel/plugin-proposal-decorators`), use a [custom .babelrc file](/docs/babel/#how-to-use-a-custom-babelrc-file).
```

Comment:
```suggestion
To add TypeScript Babel plugins (e.g. `@babel/plugin-proposal-decorators`), you can try using a [custom .babelrc file](/docs/babel/#how-to-use-a-custom-babelrc-file).
```
What do you think about making this sound more experimental? It could set it up as more of an exploration someone could take rather than being officially supported.